tests/test_serializer_nested.py

```diff
@@ -38,3 +38,88 @@ def test_nested_serialize_empty(self):
         }
         serializer = self.Serializer()
         assert serializer.data == expected_data
+
+
+class TestNestedSerializerWithMany:
+    def setup(self):
+        class PaymentInfoSerializer(serializers.Serializer):
+            url = serializers.URLField()
+            amount = serializers.DecimalField(max_digits=6, decimal_places=2)
+
+        class NestedSerializer(serializers.Serializer):
+            foo = serializers.CharField()
+
+        class AcceptRequestSerializer(serializers.Serializer):
+            ERROR_MESSAGE = '`payment_info` is required under this condition'
+
+            nested = NestedSerializer(many=True)
+            payment_info = PaymentInfoSerializer(many=True, allow_null=True)
+
+            def validate_payment_info(self, value):
```

Comment:
Let's drop this from the standard case - it's confusing.
If we do want to test this have it restricted to the single test case that deals with it.